training_step/bert/component.yaml

```diff
@@ -0,0 +1,31 @@
+name: train_pytorch_cnn
+description: Train PyTorch ImageNet CNN
```

Comment:
Please change the name and description to match the bert example

Reply:
Name changes have been done